rmltraintfbbox/rmltraintfbbox/core.py

```diff
@@ -119,78 +122,135 @@ def train(ctx: click.Context, train: TrainInput):
         experiment.log_asset(pipeline_config_path)
 
     # get number of training steps
-    num_train_steps = metadata['hyperparameters']['train_steps']
-    num_train_steps = int(num_train_steps)
-
-    tf_config = tf.estimator.RunConfig(model_dir=model_dir)
-    train_and_eval_dict = model_lib.create_estimator_and_inputs(
-        run_config=tf_config,
-        sample_1_of_n_eval_examples=1,
-        hparams=model_hparams.create_hparams(None),
-        pipeline_config_path=pipeline_config_path,
-        train_steps=num_train_steps)
+    num_train_steps = int(metadata['hyperparameters']['train_steps'])
```

Comment:
Does it matter that we cast to int if an int isn't passed in?

Reply:
it would if what got passed in couldn't be cast as an int - since we don't have any field validation right now, the `metadata['hyperparameters']['train_steps']` could be an array, or string, or anything else

not a big deal for now though. it'll just fail without good warning to the user